client/my-sites/stats/stats-notices/index.jsx

```diff
@@ -1,96 +1,119 @@
+		siteId,
+		'new_stats_feedback'
+	);
+	const { mutateAsync: postponeNewStatsFeedbackAsync } = useNoticeVisibilityMutation(
+		siteId,
 		'new_stats_feedback',
 		'postponed',
-		null,
 		30 * 24 * 3600
 	);
-	const dismissOptOutNotice = updateNewStatsFeedBackStats( 'opt_out_new_stats', 'dismissed' );
-	const dismissFeedbackNotice = updateNewStatsFeedBackStats( 'new_stats_feedback', 'dismissed' );
+
+	const onTakeSurveyClick = () => {
+		dismissNewStatsFeedbackAsync();
+		// Leave some time for the notice to be dismissed.
+		setTimeout(
+			() =>
+				( window.location.href =
+					'https://jetpack.com/redirect?source=new-jetpack-stats-usage-survey' ),
+			250
+		);
+	};
+
+	const dismissFeedbackNotice = () => {
+		setNoticeDismissed( true );
+		dismissFeedbackNoticeAsync();
+	};
+
+	const onRemindMeLaterClick = () => {
+		setNoticeDismissed( true );
+		postponeNewStatsFeedbackAsync();
+	};
+
+	const dismissOptOutNotice = () => {
+		setNoticeDismissed( true );
+		dismissOptOutNoticeAsync();
+	};
 
 	return (
-		<>
-			{ showOptOutNotice && (
-				<div className="inner-notice-container has-odyssey-stats-bg-color">
-					<NoticeBanner
-						level="success"
-						title={ translate( 'Welcome to the new Jetpack Stats!' ) }
-						onClose={ dismissOptOutNotice }
-					>
-						{ translate(
-							'{{p}}Enjoy a more modern and mobile friendly experience with new stats and insights to help you grow your site.{{/p}}{{p}}If you prefer to continue using the traditional stats, {{manageYourSettingsLink}}manage your settings{{/manageYourSettingsLink}}.{{/p}}',
-							{
-								components: {
-									p: <p />,
-									manageYourSettingsLink: (
-										<a href="/wp-admin/admin.php?page=jetpack#/settings?term=stats" />
-									),
-								},
-							}
-						) }
-					</NoticeBanner>
-				</div>
-			) }
-			{ showFeedbackNotice && (
-				<div className="inner-notice-container has-odyssey-stats-bg-color">
-					<NoticeBanner
-						level="info"
-						title={ translate( "We'd love to hear your thoughts on the new Stats" ) }
-						onClose={ dismissFeedbackNotice }
-					>
-						{ translate(
-							"{{p}}Now that you've gotten familiar with the new Jetpack Stats, we'd love to hear about your experience so we can continue to shape Jetpack to meet your needs.{{/p}}{{p}}{{takeSurveyButton}}Take quick survey{{/takeSurveyButton}} {{remindMeLaterLink}}Remind me later{{/remindMeLaterLink}}{{/p}}",
-							{
-								components: {
-									p: <p />,
-									takeSurveyButton: (
-										<button
-											type="button"
-											className="notice-banner__action-button"
-											onClick={ onTakeSurveyClick }
-										/>
-									),
-									remindMeLaterLink: (
-										<button
-											className="notice-banner__action-link"
-											onClick={ onRemindMeLaterClick }
-										/>
-									),
-								},
-							}
-						) }
-					</NoticeBanner>
-				</div>
-			) }
-		</>
+		! noticeDismissed && (
```

Comment:
```suggestion
		! noticeDismissed && isOdysseyStats && (
```

Could we check the isOdysseyStats earlier?

Reply:
The reason I put it this way is that we are going to add notice for Calypso Stats too, and `isOdyssey` would then be moved down 😃